styles/colors-base.less

```
@@ -439,6 +439,14 @@
 @agate-switch-selected-color: inherit;
 @agate-switch-selected-focus-bg-color: @agate-switch-selected-bg-color;
 
+// SwitchItem
+// ---------------------------------------
+@agate-switchitem-label-color: inherit;
+@agate-switchitem-label-focus-selected-color: inherit;
+@agate-switchitem-label-selected-color: inherit;
+@agate-switchitem-selected-border-color: transparent;
+@agate-switchitem-focused-selected-border-color: trasnaprent;
```

```suggestion
@agate-switchitem-focused-selected-border-color: @agate-item-selected-border-color;;
```
Also, delete the definition from silicon below as this will cover silicon as well